packages/meta/README.md

```diff
@@ -51,16 +55,10 @@ Notably these features are missing:
 > To provide feedback please [open an issue](https://github.com/temporalio/sdk-node/issues) and feel free to
 > talk to us on the [community slack channel](https://slack.com/app_redirect?app=TNWA8QCGZ&channel=nodejs-sdk).
 
-### Installation
+## Installation
 
 See the [getting started](https://docs.temporal.io/docs/node/getting-started) guide for setting up a project on your laptop.
 
-### Documentation
```

Comment:
Why don't we move this section up instead of this entire restructure?

Reply:
I find this list of links:

![image](https://user-images.githubusercontent.com/251288/128636916-dd41b30d-f463-4ea6-89be-bd97f8f63a12.png)

to be clearer/quicker to understand than this section:

![image](https://user-images.githubusercontent.com/251288/128636936-2e987a9a-b8ee-411e-acb2-bc12b1257da7.png)

I raised the heading size because Features was so small I didn't realize it was a section:

![image](https://user-images.githubusercontent.com/251288/128637113-b7c41a11-7a93-4deb-889a-62d69d9af43d.png)

Reply:
Okay, I'm fine with the links, I'd just rather move them below the first paragraph explaining what temporal is and putting them in their own section with a `## Documentation` header.
Might as well make them into list items as well.